page-objects/lib/alert.po.ts

```diff
@@ -16,4 +17,19 @@ export class AlertPO extends BasePO {
 	get locatorCloseButton(): Locator {
 		return this.locatorRoot.locator(this.selectors.closeButton);
 	}
+
+	async state() {
+		return this.locatorRoot.evaluate(async (rootNode: HTMLElement, selectors) => {
+			await Promise.all(rootNode.getAnimations().map((animation) => animation.finished));
```

Comment:
I didn't know this api, but are you sure it's usefull in our case too ? I'm curious to know if it also works with transitions (what we use), or only with animations...

Reply:
in fact after we have modified the 'fade' animation it is not needed anymore as fade always stays. The issue was that the state was returned before the fade was removed, so we needed animation.finished.
Don't know whether it works with transitions though - it is not specified in https://developer.mozilla.org/en-US/docs/Web/API/Element/getAnimations